merge_and_rename_NGI_fastq_files.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env/python
+# Merges all fastq_files from each samples into one file. 
+# ARGV 1 Base directory with files to be merged
+# ARGV 2 Destination dir of the merged output files
+
+import re
+import os
+import sys
+import shutil
+def merge_files(dest_dir, fastq_files):
+    #Match NGI sample number from flowcell
+    sample_pattern=re.compile("^(.+)_S[0-9]+_.+_R([1-2])_")
```

Comment:
This re is a bit too general maybe,  you capture the complete path but you might end up in matching this:
``` /PATH_TO_SOMEWHERE/SILLY_FOLDER_S0_SILLY/FASTQ_S1_R1_SOMETHING.fastq.gz```
and group 1 would be 
``` /PATH_TO_SOMEWHERE/SILLY_FOLDER```
which later on will create troubles.
It is safer to extract the file name `os.path.basename`, match it (maybe using also the fact that it needs to look like `^P[0-9]+_[0-9]+_S....` and then extract the matching section and recompose the new file name